frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html

```diff
@@ -50,18 +51,20 @@ <h2>Project members</h2>
           "
         >
           <div class="flex">
-            <mat-icon class="accountCircle" mat-list-icon
-              >account_circle</mat-icon
-            >
+            <div class="flex items-center mr-4 ml-0">
+              <mat-icon class="text-3xl !h-8 !w-8 my-auto" mat-list-icon
+                >account_circle</mat-icon
+              >
+            </div>
             <div>
-              <div class="username">{{ user.user.name }}</div>
-              <div class="userrole">
+              <div class="text-xl break-all">{{ user.user.name }}</div>
```

Comment:
```suggestion
              <div class="text-[1.25em] break-all">{{ user.user.name }}</div>
```

The username is now way larger than before, taking a lot more space.